code/modules/mob/living/carbon/carbon_update_icons.dm

```diff
@@ -384,8 +384,7 @@
 /obj/item/bodypart/proc/generate_husk_key()
 	RETURN_TYPE(/list)
 	. = list()
-	. += "[husk_type]"
-	. += "-husk"
+	. += "husk"
 	. += "-[body_zone]"
```

Comment:
This needs `species_id` now.

Reply:
Hm... Actually, having thought about this more, keeping the `husk_type` variable but just moving the sprites into their own `husk.dmi` file might be a better option than what I'm doing currently.
Using `species_id`/`limb_id` so that it's the same as standard limb icon states (e.g `lizard_husk_l_arm`) would work as well, but currently every species other than Teshari is just using the same human husk sprite, so structuring it like that would result in every `\[species]\bodyparts.dmi` file having the same 8 icon states copy pasted into it.

Reply:
var/icon_husk is a thing. they can all point to the human one. I niw remember why i did this.

Reply:
Okay, the latest commit uses pretty much the opposite approach but it seems to be the cleanest solution.
Everything husk related is contained in `icons/mob/husk.dmi`, with bodyparts setting `husk_type` to choose different icon sets. (Currently `"humanoid"` and `"teshari"`)

Reply:
So would you like me to change this, or..?
`icon_husk` is set to the human sprite by default already, with the Teshari bodyparts overriding it.